tests/ssl-opt.sh

```diff
@@ -204,6 +204,17 @@ requires_config_value_at_most() {
     fi
 }
 
+requires_config_value_exactly() {
+    VAL=$( get_config_value_or_default "$1" )
+    if [ -z "$VAL" ]; then
+        # Should never happen
+        echo "Mbed TLS configuration $1 is not defined"
+        exit 1
+    elif [ "$VAL" -eq "$2" ]; then
```

Comment:
This condition should be negated, shouldn't it?

Reply:
Thanks @hanno-arm  for finding this, fixed in the next commit.